dash/include/dash/allocator/internal/Types.h

```diff
@@ -0,0 +1,133 @@
+#ifndef DASH__ALLOCATOR__INTERNAL__TYPES_H__INCLUDED
+#define DASH__ALLOCATOR__INTERNAL__TYPES_H__INCLUDED
+#include <cstddef>
+#include <dash/dart/if/dart_globmem.h>
+#include <ostream>
+#include <tuple>
+
+namespace dash {
+namespace allocator {
+
+template <typename LPtr, typename GPtr = dart_gptr_t>
+struct allocation_rec {
```

Comment:
You are right. It is actually only an internal type (internal namespace) and will be removed soon. It is used only in the EpochSynchronizedAllocator. And this will be changed to follow the new memory space concept in another pull request (near future). At the moment, only static containers are adapted to it. And this was a significant amount of work.